ansible/ansible.cfg

```diff
@@ -0,0 +1,10 @@
+[defaults]
+remote_user = ubuntu
+retry_files_enabled = False
+
+[privilege_escalation]
+become = True
+
+# [ssh_connection]
```

Comment:
Did you mean to keep these commented lines in the config for future reference, or can they be cleaned up?